MerlinAU.sh

```diff
@@ -2509,8 +2591,20 @@ Please manually update to version $minimum_supported_version or higher to use th
     "$isInteractive" && printf "\nRestarting web server... Please wait.\n"
     /sbin/service restart_httpd && sleep 5
 
-    _EntwareServicesHandler_ stop
+	#Send last email notification before flash
     _SendEMailNotification_ START_FW_UPDATE_STATUS
+	
+    # Check if '/opt/bin/diversion' exists
+    if [ -f /opt/bin/diversion ]; then
+	#Stop Divsersion services before flash
+        Say "Stopping Diversion service..."
+        /opt/bin/diversion unmount
+	else
+        Say "Diversion service not installed."
```

Comment:
I don't think we need to tell users that they don't have Diversion. It's not a requirement to have it installed.